src/cpu.c

```diff
@@ -21,3 +22,10 @@ void resetCpu(struct CPU * cpu, struct BUS * bus) {
   cpu->status.overflow = false;
   cpu->status.negative = false;
 }
+
+void clockCpu(struct CPU *cpu, struct BUS *bus) {
+  // TODO: Do the emulation xD
+  unsigned char current = readBus(cpu->pc, bus);
```

Comment:
Clock cpu means a single clock or a single instruction execution with a lot of cycle clocks?
Just asking to know whether we should increment the PC here or not.
Thinking at hardware perspective you know way more than me, is that CPU responsibility to increment the PC or when Bus is readed, its kind of atomic operation?

Reply:
Exactly, is more like one instruction cycle, thinks like address mode and opcodes can move pc around

Reply:
So, probably this pc++ will be moved when we start o interpret the program flow and execution?

Reply:
I'm the other PR #17  will be clearer why I do this pc++ there